src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1560,6 +1560,44 @@ def test_webapp_create_linux_free(self, resource_group):
         self.cmd('webapp config show -g {} -n {}'.format(resource_group, webapp_name)).assert_with_checks([
             JMESPathCheck('alwaysOn', False)])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_webapp_linux_free')
```

Comment:
this is not a test you are adding was this removed & you are re-adding? Or is this being copied over again?